rm-main/src/app.rs

```diff
@@ -45,7 +45,7 @@ impl Ctx {
             }
             Err(_) => {
```

Comment:
Err(e) => {
in order to not to ignore it as it may contain useful information for the user